e2e/jumpbox.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: jumpbox
```

Comment:
why do we not use `az vmss run-command` to collect logs but with jumpbox? 
The node provisioning may fail before installing sshserver.

Reply:
I don't know how to set the command of powershell. When I use Get-Content, the length of return message is limited and I can only get the last several lines of the log file.

Reply:
we can use `az vmss run-command` to upload the logs to a pre-created storage account